Task_1/Parts/Pupil.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_1
+{
+    abstract class Pupil
+    {
+        private string studentState;
+        private string studentName;
+
+        public string StudentState
+        {
+            get => studentState;
+            set => studentState = value;
+        }
+
+        public string StudentName
+        {
+            get => studentName;
+            set => studentName = value;
+        }
```

Comment:
As you don't do any additional checks then use auto properties 
```suggestion
    public string StudentState {get;set}
    public string StudentName {get;set;}
```